### PR TITLE
Some work to reduce JS bundle size

### PR DESCRIPTION
**Pull request by longhotsummer**
* significantly reduce JS bundle size by about 1MB
* remove moment - unnecessary
* remove large lodash library

Before:

```
npx webpack --stats

[@vue/compiler-sfc] compileTemplate now requires the `id` option.`.

asset app-prod.js 3.17 MiB [emitted] (name: app)
asset vendors-node_modules_stencil_core_internal_client_shadow-css_js.app-prod.js 17.1 KiB [compared for emit] (id hint: vendors)
runtime modules 7.08 KiB 11 modules
orphan modules 689 bytes [orphan] 9 modules
modules by path ./node_modules/ 2.24 MiB
  cacheable modules 2.24 MiB 413 modules
  ./node_modules/@stencil/core/internal/client/ lazy ^\.\/.*\.entry\.js.*$ include: \.entry\.js$ exclude: \.system\.entry\.js$ namespace object 160 bytes [built] [code generated]
  ./node_modules/moment/locale/ sync ^\.\/.*$ 3.21 KiB [optional] [built] [code generated]
modules by path ./peachjam/js/ 259 KiB
  modules by path ./peachjam/js/components/ 227 KiB 149 modules
  optional modules 11.1 KiB [optional] 8 modules
  modules by path ./peachjam/js/*.ts 6.85 KiB 4 modules
  modules by path ./peachjam/js/utils/*.ts 5.27 KiB
    ./peachjam/js/utils/vue-utils.ts 507 bytes [built] [code generated]
    ./peachjam/js/utils/function.ts 4.77 KiB [built] [code generated]
  ./peachjam/js/items.json 1.23 KiB [built] [code generated]
  ./peachjam/js/dom.js 7.22 KiB [built] [code generated]
webpack 5.76.0 compiled successfully in 9606 ms
```

after:

```
npx webpack --stats
[@vue/compiler-sfc] compileTemplate now requires the `id` option.`.

asset app-prod.js 2.3 MiB [emitted] (name: app)
asset vendors-node_modules_stencil_core_internal_client_shadow-css_js.app-prod.js 17.1 KiB [compared for emit] (id hint: vendors)
runtime modules 6.96 KiB 10 modules
orphan modules 689 bytes [orphan] 9 modules
javascript modules 1.73 MiB
  modules by path ./node_modules/ 1.5 MiB 167 modules
  modules by path ./peachjam/js/ 239 KiB 156 modules
json modules 10.8 KiB
  modules by path ./peachjam/js/locale/en/*.json 2.71 KiB 2 modules
  modules by path ./peachjam/js/locale/fr/*.json 3 KiB 2 modules
  modules by path ./peachjam/js/locale/sw/*.json 2.84 KiB 2 modules
  ./peachjam/js/locale/pt/translation.json 2.21 KiB [optional] [built] [code generated]
webpack 5.76.0 compiled successfully in 9037 ms
```

Ref #1318

### CODE REVIEW

File: peachjam/js/components/pdf-renderer.ts [hunk 1]
```diff
@@ -223,24 +223,6 @@ class PdfRenderer {
       this.previewPanelsContainer.appendChild(panelPreview);
     }
   }
-
-  decoratePdf () {
```
Comment (longhotsummer) [inline peachjam/js/components/pdf-renderer.ts:227]: This is unused.

**Comment by longhotsummer** [comment]
Oops, my stats were for development mode compilation. Here are the stats for production:

Before: 893KB
After: 584KB

**Comment by actlikewill** [comment]
> Awesome. I can see lodash.debounce there. Is that in use?

nvm, I can see it the pdf renderer

**Comment by longhotsummer** [comment]
We could probably pull lodash.debounce out, but it least it's not pulling in a million other lodash things. I'll let sandrava play with that.

**Pull request merged**